membership-attribute-service/conf/DEV.public.conf

```diff
@@ -24,6 +24,11 @@ play.filters.cors.allowedOrigins = [
   "https://thegulocal.com"
 ]
 
+authentication {
```

Comment:
don't bother changing it, but for a single value you cn just do 
`authentication.key="a-nice-secret"`
all on one line